src/components/ProcessCreate/Census/select-components.tsx

```diff
@@ -72,10 +73,12 @@ const OptionToken = (props: OptionProps<any, false, GroupBase<any>>) => {
   } else {
     return (
       <chakraComponents.Option {...props}>
-        <Flex alignItems='center' gap={2}>
-          <CryptoAvatar name={name} icon={iconURI} id={ID} chainId={chainID} />
-
-          <Text>{children}</Text>
+        <Flex justifyContent={'space-between'} alignItems={'center'} w='full'>
+          <Flex gap={2} color='black'>
+            <CryptoAvatar name={name} icon={iconURI} id={ID} chainId={chainID} />
+            <Text>{children}</Text>
+          </Flex>
+          {isSelected && <CheckIcon color='input.dropdown.check_icon' ml='auto' />}
```

Comment:
I'm wondering... how were we previously marking the selected items?

Reply:
With a colored background